.github/main.yml

```diff
@@ -0,0 +1,22 @@
+name: GitHub Actions Demo
+on: [push]
```

Comment:
currently there are no jobs or checks executed for this PR, review if this `on` trigger needs to be changed by a:
`on: [pull_request]`. 

Also review that you have enabled the Require status check option in the branch protection rules:
![image](https://user-images.githubusercontent.com/10209180/180567494-2996fe5e-ead3-405d-b9d6-9ea18f9b9cfe.png)